server/monitor/src/main/java/org/apache/accumulo/monitor/Monitor.java

```diff
@@ -460,15 +460,11 @@ public void run() {
     log.info("stop requested. exiting ... ");
   }
 
-  private ServletHolder getDefaultServlet() {
-    return new ServletHolder(new DefaultServlet() {
-      private static final long serialVersionUID = 1L;
-
-      @Override
-      public Resource getResource(String pathInContext) {
-        return Resource.newClassPathResource("/org/apache/accumulo/monitor" + pathInContext);
-      }
-    });
+  private ServletHolder getDefaultServlet(ServletContextHandler context) {
```

Comment:
@ctubbsii - might need your assistance here. The Monitor server is not serving up the static files. I've looked at several online resources, not sure what to try next.